src/imagetilesource.js

```diff
@@ -171,9 +171,7 @@
          * @param {Number} level
          * @param {OpenSeadragon.Point} point
          */
-        getTileAtPoint: function (level, point) {
-            return new $.Point(0, 0);
-        },
+        getTileAtPoint: $.TileSource.prototype.getTileAtPoint,
```

Comment:
Actually you can just get rid of this line entirely; these subclasses inherit everything from TileSource unless overridden. Don't forget to remove the doc comment as well.